csharp/src/Ice/InstrumentationI.cs

```diff
@@ -345,16 +336,9 @@ public string GetId()
             return null;
         }
 
-        public Connection? GetConnection() => _current.Connection;
+        public string GetEndpointString() => GetEndpoint() == null ? "" : GetEndpoint()!.ToString();
```

Comment:
Same here, I don't see why it's necessary.

Reply:
Removed.